Catappult/appcoins-wallet-android
# [feature/APPC-3224] Notify users becoming vip

**What does this PR do?**

   adds different notification to user becoming vip, and adds new screen after notification click

**Database changed?**

    No

**Where should the reviewer start?**

- AlmostVipFragment.kt
- PerkBonusAndGamificationService.kt

**How should this be manually tested?**

  - reach near the end of a level 7 (last 2k of that level or something)
  - make a purchase
  - click the notification
  - check the new screen and button behaviour

**What are the relevant tickets?**

 https://aptoide.atlassian.net/browse/APPC-3224

**Code Review Checklist**
